framework/src/modules/chain/components/storage/entities/migration.js

```diff
@@ -17,9 +17,11 @@
 const path = require('path');
 const fs = require('fs-extra');
 const { defaults, pick } = require('lodash');
-const { NonSupportedOperationError } = require('../errors');
-const filterType = require('../utils/filter_types');
-const BaseEntity = require('./base_entity');
+const {
+	NonSupportedOperationError,
+} = require('../../../../../components/storage/errors');
+const filterType = require('../../../../../components/storage/utils/filter_types');
+const BaseEntity = require('../../../../../components/storage/entities/base_entity');
```

Comment:
Ideally chain module should not access component files directly.

I know it still doesn't solve the problem, but can we export `BaseEntity`, `Utils` and `Errors` from `framework/src/components/storage/index.js` so that we have a predefined interface?

Reply:
@nazarhussain do you agree as I do?

Reply:
@lsilvs FYI https://github.com/LiskHQ/lisk/issues/2945